int32 id
int32[] goal
---
bool success